product_order_noname/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Vauxoo, S.A. de C.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Product Order No Name',
+    'summary': 'Speedup product retrieve',
+    "version": "12.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Technical Settings",
+    "website": "https://github.com/OCA/product-attribute",
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "maintainers": ["WR-96", "moylop260"],
+    "license": "AGPL-3",
```

Comment:
LGPL

Reply:
[AGPL](https://github.com/OCA/maintainer-tools/blob/f1b559574648c01558b543e7550f5b9be7c92784/template/module/__manifest__.py#L14)

Reply:
I don't understand that as a required license, just the default one.